libs/game/sprite.ts

```diff
@@ -237,6 +309,8 @@ class Sprite extends sprites.BaseSprite {
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._vy)); k += 2;
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._ax)); k += 2;
         buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._ay)); k += 2;
+        buf.setNumber(NumberFormat.Int16LE, k, Fx.toInt(this._sx)); k += 2;
```

Comment:
I think you need to increase the +12 to +16 above?

Reply:
good catch!